index.html

```diff
@@ -1929,224 +1974,71 @@ <h2>
 automatically typed to provide stronger type guarantees for use cases that
 require them.
         </li>
-        <li>
```

Comment:
Please respond to all comments folks raise on your PRs, even if you're copying/pasting... otherwise, I have no idea what action you took (because I may have missed the resolved conversation where you said you took an action). In general... don't resolve conversations you didn't start, and always respond to every comment on PRs so the other party knows you took action (or decided to not take action).